src/Token.php

```diff
@@ -43,10 +43,10 @@ public function __construct(OAuth2\Server $server)
      */
     public function __invoke(ServerRequestInterface $request, ResponseInterface $response, array $arguments = [])
     {
-        return ResponseBridge::fromOAuth2(
```

Comment:
I think this can be done simply with a `withHeader` call.

```php
return ResponseBridge::fromOAuth2(
    $this->server->handleTokenRequest(RequestBridge::toOAuth2($request))
)->withHeader('Content-Type', 'application/json');
```